src/graphql/subscriptions/patients.ts

```diff
@@ -0,0 +1,79 @@
+import gql from 'graphql-tag';
+
+export const PATIENT_UPDATED = gql`
+  subscription OnPatientUpdated($eventId: ID!) {
```

Comment:
we should keep `camelCase` for graqhql operations
```suggestion
  subscription onPatientUpdated($eventId: ID!) {
```